src/main/java/seedu/address/model/widget/WidgetObject.java

```diff
@@ -112,31 +119,22 @@ public String textTwo() {
         return divs[4] == null ? "" : divs[4];
     }
 
-    /**
-     * Getter for 'attributes'.
-     *
-     * @return Div3.
-     */
-    public String divThree() {
-        return divs[5] == null ? "" : divs[5];
-    }
-
     /**
      * Getter for 'attributes'.
      *
      * @return Text3.
      */
     public String textThree() {
```

Comment:
Might make the code more readable if it is something like `getTextThree()`, `getDiv4()` etc